package.json

```diff
@@ -97,6 +97,8 @@
     "storybook": "storybook dev -p 6006",
     "storybook:build": "storybook build",
     "test:unit": "NODE_ENV=development vitest run",
+    "test:sails": "NODE_ENV=development vitest run --config tests/unit/vitest-sails.config.mjs",
+    "test": "npm run test:unit && npm run test:sails",
```

Comment:
`test:unit` is only for pure unit tests now, `test:sails` is for the sails specific tests and `test` will now run both.